sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -367,17 +375,21 @@ def init_ports_status_for_y_cable(platform_sfp, platform_chassis, y_cable_presen
     config_db, state_db, port_tbl, y_cable_tbl = {}, {}, {}, {}
     static_tbl, mux_tbl = {}, {}
     port_table_keys = {}
+    xcvrd_log_tbl = {}
 
     y_cable_platform_sfputil = platform_sfp
     y_cable_platform_chassis = platform_chassis
 
+    fvs_updated = swsscommon.FieldValuePairs([('enable_log', 'false')])
```

Comment:
we can go with `verbose` or `log_verbosity` in case we are going to have various log verbosity.

Reply:
fixed